src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -1547,6 +1547,70 @@ hibernate.connection.url=jdbc:hsqldb:mem:rm;hsqldb.tx=mvcc;hsqldb.lob_file_scale
 
 To switch to a standalone database, see <<_perf_tuning_db>>.
 
+== Nodes and Task Recovery
+
+While your tasks are running on nodes, the scheduler can experience a transient failure and come back to life without
+losing the on-going computations. This is possible thanks to the nodes and task recovery mechanism run by the scheduler.
+When the scheduler is starting, it checks the nodes and the tasks that were persisted in database to be able to recover
+on going tasks and alive nodes. For each node that is supposed to be handled by the scheduler, the recovery will consist
+into trying to contact these nodes. If a node is alive, the reference that is retrieved with the lookup of this node is
+kept by the resource manager and the internal data structures of the scheduler are updated with this node information.
+If a persisted node is not alive at recovery time, then the node will be recreated automatically.
+
+=== Scope
+
+A recovery is not a restart, and as such a clean restart will not preserve on-going tasks and running nodes. Indeed, if
```

Comment:
In this paragraph, I would rather see, a short use case, what would happen without this feature, what does this feature brings